src/etc/test-float-parse/runtests.py

```diff
@@ -195,9 +195,9 @@ def main():
     global MAILBOX
     tests = [os.path.splitext(f)[0] for f in glob('*.rs')
                                     if not f.startswith('_')]
-    whitelist = sys.argv[1:]
-    if whitelist:
-        tests = [test for test in tests if test in whitelist]
+    listed = sys.argv[1:]
```

Comment:
`listed` sound like a bool to me. Could we change it to `args` or `arguments` ?

Reply:
yea listed is abit weird word here args seems reasonable.